README.md

```diff
@@ -67,24 +90,45 @@ for dataset in client.list_remote_datasets():
     print(dataset.slug, dataset.image_count)
 ```
 
-#### Upload data to a [remote] project (images/videos)
 
-Uploads data to an existing remote project.
-It takes the dataset slug and a list of file names of images/videos to upload as parameters.
+#### Create/remove a dataset 
+
+Dataset creation is handled by the client:
 
 ```python
 from darwin.client import Client
 
 client = Client.default()
-dataset = client.get_remote_dataset(slug="example-dataset")
-progress = dataset.upload_files(["test.png", "test.mp4"])
-for _ in progress():
-    print("file uploaded")
+client.create_dataset(name="This Is My New Dataset")
 ```
 
-#### Pull a [remote] project
+Whereas dataset removal is handled directly by the dataset themselves:
+
+```python
+from darwin.client import Client
 
-Downloads a remote project, images and annotations, in the projects directory (specified in the authentication process [default: ~/.darwin/projects]).
+client = Client.default()
+dataset = client.get_remote_dataset(slug="this-is-my-new-dataset")
+dataset.remove()
```

Comment:
Does this remove the local data or delete the project on the server?

Reply:
Archives the remote with this:
```
    def remove(self):
        """Archives (soft-deletion) the remote dataset"""
        self.client.put(f"projects/{self.project_id}/archive", payload={})
```

Reply:
If I already did `pull` (so it exists in both local and remote) it seems a bit counter-intuitive that `remove` would only affect the remote dataset. I see `pull` and `push` as functions referenced to "local" so I would assume that `remove` would act in the same way.

Reply:
If you did `pull()` a dataset the design choice to remove the local version is currently `rm -rf PATH`, (might be user OS specific, this is just an example). This mirrors the behaviour of browsers for example. 

All dataset functionality is related to the remote version of it, because that is where darwin is - or at least I understand it like this :)